src/cpp/cpp_convert_type.cpp

```diff
@@ -28,7 +28,7 @@ class cpp_convert_typet
   unsigned unsigned_cnt, signed_cnt, char_cnt, int_cnt, short_cnt,
            long_cnt, const_cnt, restrict_cnt, constexpr_cnt, volatile_cnt,
            double_cnt, float_cnt, complex_cnt, cpp_bool_cnt, proper_bool_cnt,
-           extern_cnt, wchar_t_cnt, char16_t_cnt, char32_t_cnt,
+           extern_cnt, noreturn_cnt, wchar_t_cnt, char16_t_cnt, char32_t_cnt,
```

Comment:
As an aside - variable names should not be abbreviated, though I don't know whether it is better to un-abbreviate this new one whilst leaving the old as they are or doing a separate PR to fix them all.